### ISSUE TITLE
[Feature Request] Add the ability to selectively disable items in the context menu.

### ISSUE THREAD

**Issue by sn260591** [sienori/simple-translate#5]

**Comment by NgYueHong** [comment]
Yeah, I wish to able to remove "Translate this page" from the context menu via option page. Because it is a feature that I rarely use. Having it there is annoying and may accidentally click on it.

**Comment by sienori** [comment]
I will implement this option in the future.

**Comment by jmsosso** [comment]
I also would like to have just "Translate selected text" in my context menu without having to move to the submenu popup to find it :)

**Comment by maxoku** [comment]
Yeah quickly translating selected text without having to go through another menu where I can accidently click on translating the entire page would be nice. It's not very comfortable right now :/

**Comment by maxoku** [comment]
Over 4 years from creating the thread and still no progress? :/
